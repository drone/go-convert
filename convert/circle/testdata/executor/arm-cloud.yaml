# https://circleci.com/docs/executor-intro/#arm

version: 2.1

jobs:
  build-medium:
    machine:
      image: ubuntu-2004:202101-01
    resource_class: arm.medium
    steps:
      - run: uname -a
      - run: echo "Hello, Arm!"

  build-large:
    machine:
      image: ubuntu-2004:202101-01
    resource_class: arm.large
    steps:
      - run: uname -a
      - run: echo "Hello, Arm!"

workflows:
  build:
    jobs:
      - build-medium
      - build-large
