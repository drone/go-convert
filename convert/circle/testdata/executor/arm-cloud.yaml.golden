kind: pipeline
spec:
  stages:
  - name: build-medium
    spec:
      platform:
        arch: arm64
        os: linux
      runtime:
        spec: {}
        type: cloud
      steps:
      - spec:
          run: |-
            set -eo pipefail
            uname -a
          shell: bash
        type: script
      - spec:
          run: |-
            set -eo pipefail
            echo "Hello, Arm!"
          shell: bash
        type: script
    type: ci
  - name: build-large
    spec:
      platform:
        arch: arm64
        os: linux
      runtime:
        spec: {}
        type: cloud
      steps:
      - spec:
          run: |-
            set -eo pipefail
            uname -a
          shell: bash
        type: script
      - spec:
          run: |-
            set -eo pipefail
            echo "Hello, Arm!"
          shell: bash
        type: script
    type: ci
version: 1
